playbook/app/pb_kits/playbook/pb_checkbox/docs/_checkbox_disabled.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import Checkbox from '../_checkbox'
+
+const CheckboxDisabled = (props) => {
+  return (
+    <div>
+      <Checkbox
+          disabled
+          name="default name"
+          tabIndex={0}
+          text="Disabled unchecked"
+          value="default value"
+          {...props}
+      />
+      <br />
```

Comment:
Can you use globals instead of break tags?

Reply:
Hi @jasoncypret - thanks for commenting this. While exploring an alternative solution I appreciated learning about how we are trying to get away from the pattern of using break tags in doc examples. I've come up two workable alternatives that aren't quite global props but do avoid break tags, and was wondering if you had a preference:
1. Surrounding each individual Radio/Checkbox item in it's own div was enough to get them on different lines.
2. Adding flex with column orientation to the surrounding container.
Please let me know which makes the most sense and I'll get that change pushed up, thank you!